youtube_dl/extractor/ora.py

```diff
@@ -6,13 +6,14 @@
 from ..compat import compat_urlparse
 from ..utils import (
     get_element_by_attribute,
+    js_to_json,
     qualities,
     unescapeHTML,
 )
 
 
 class OraTVIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?ora\.tv/([^/]+/)*(?P<id>[^/\?#]+)'
+    _VALID_URL = r'https?://(?:www\.)?(ora\.tv|unsafespeech\.com)/([^/]+/)*(?P<id>[^/\?#]+)'
```

Comment:
There's a capturing group right after the backlash... is it really such a huge problem?